src/sinks/elasticsearch/sink.rs

```diff
@@ -91,7 +97,9 @@ where
     }
 }
 
-pub fn process_log(
+/// Any `None` values returned from this function will already result in a `TemplateRenderingError`
+/// being emitted, so no further `discarded_event` needs emitting.
```

Comment:
Not really an issue but I've been trying to refer to it as  " `EventsDropped` internal event " , in comments to correspond to the instrumentation spec doc ?